zcash_primitives/src/consensus.rs

```diff
@@ -365,6 +365,10 @@ impl Parameters for Network {
             Network::TestNetwork => TEST_NETWORK.b58_script_address_prefix(),
         }
     }
+
+    fn is_nu_active(&self, nu: NetworkUpgrade, height: BlockHeight) -> bool {
```

Comment:
Why is this override of the default impl necessary?

Reply:
uhmmm this doesn't seem to be in the right place!  great catch! I'll look into it